File: 2037-minimum-number-of-moves-to-seat-everyone/2037-minimum-number-of-moves-to-seat-everyone.js
/**
 * @param {number[]} seats
 * @param {number[]} students
 * @return {number}
 */
var minMovesToSeat = function(seats, students) {
  const comp = (a, b) => a - b
  seats.sort(comp)
  students.sort(comp)
  let ans = 0
  for (let i = 0; i < seats.length; i ++) {
    ans += Math.abs(seats[i] - students[i])
  }
  return ans
};

File: 2037-minimum-number-of-moves-to-seat-everyone/2037-minimum-number-of-moves-to-seat-everyone.go
func minMovesToSeat(seats []int, students []int) int {
  sort.Ints(seats)
  sort.Ints(students)
  ans := 0
  for i := 0; i < len(students); i++ {
    if students[i] > seats[i] {
      ans += students[i] - seats[i]
    } else {
      ans += seats[i] - students[i]
    }
  }
  return ans
}